druid/src/ext_event.rs

```diff
@@ -117,6 +118,25 @@ impl ExtEventSink {
         ));
         Ok(())
     }
+
+    /// Schedule an idle callback.
+    pub fn add_idle_callback<T: 'static + Data>(&self, cb: impl FnOnce(&mut T) + Send + 'static) {
+        let mut handle = self.handle.lock().unwrap();
+        if let Some(handle) = handle.as_mut() {
+            handle.add_idle(|win_handler| {
+                let win_handler = win_handler
+                    .as_any()
+                    .downcast_mut::<DruidHandler<T>>()
+                    .unwrap_or_else(|| {
+                        panic!(
```

Comment:
Maybe `debug_panic!`?

Reply:
yeah makes sense to skip running the callback and don't panic the UI thread